dev/com.ibm.ws.cdi.third.party_fat/fat/src/com/ibm/ws/cdi/thirdparty/tests/HibernateSearchTest.java

```diff
@@ -57,24 +58,23 @@
 
 */
 
+@SkipForRepeat(EERepeatActions.EE7_ID)
 @RunWith(FATRunner.class)
 public class HibernateSearchTest extends FATServletClient {
```

Comment:
This test never ran on EE7.

The old configuration said "run in the original config, then EE8, then EE9". However, the server.xml for this server was set up for EE8 which meant the test ran on EE8 twice.